core/templates/dev/head/base.html

```diff
@@ -224,7 +224,7 @@
                     </div>
                   {% endif %}
 
-                  <div class="collapse navbar-collapse ng-cloak">
+                  <div class="ng-cloak">
```

Comment:
Yeah it does, silly of me I didn't notice. Actually it collapses the navbar if width is less than 768, do we need to editor for that small devices?